### PR TITLE
update golangcilint

### PR DESCRIPTION
**Pull request by pierrebeitz**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -34,7 +34,7 @@ vet: lint
 
 .PHONY: lint
 lint: docker-image
-	$(call inDocker, golangci-lint run)
+	echo "test"
```
Comment (janisz) [inline Makefile:37]: ?

File: ci/release.py [hunk 2]
```diff
@@ -14,23 +14,23 @@
     artifacts = [
         ("linux/dcos",       "cli/releases/binaries/dcos/linux/x86-64/latest/dcos"),
         ("darwin/dcos",      "cli/releases/binaries/dcos/darwin/x86-64/latest/dcos"),
-        ("darwin/dcos.zip",      "cli/releases/binaries/dcos/darwin/x86-64/latest/dcos.zip"),
```
Comment (janisz) [inline ci/release.py:17]: Please change it in another PR

**Comment by pierrebeitz** [comment]
@janisz I'm currently playing around with stuff on CI and don't know how to not notify the codeowners none of the 2 prs is meant to be reviewed yet. Any idea?

**Comment by janisz** [comment]
One pr is opened codeowners are assigned automatically. I think we can't disable it

**Comment by janisz** [comment]
I think we can close it? Windows builds were fixed.

**Pull request closed without merging**